src/vs/workbench/browser/parts/statusbar/statusbarPart.ts

```diff
@@ -51,8 +56,11 @@ interface IStatusbarViewModelEntry {
 	alignment: StatusbarAlignment;
 	priority: number;
 	container: HTMLElement;
+	labelContainer: HTMLElement;
 }
 
+const CONTEXT_STATUS_BAR_FOCUSED = new RawContextKey<boolean>('statusBarFocused', true);
```

Comment:
The default should be `false`, no?

Reply:
Yes, I changed this for testing and forgot to turn it back to `false`!